wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1331,7 +1331,7 @@ def keyboard_shortcuts_dialog(context):
 
     return {
         "shortcuts": {
-            ("actions-common", _("Common actions")): [
+            ("text-editing", _("Text Editing")): [
```

Comment:
Sorry to nitpick, but for consistency, this should be sentence case.

```suggestion
            ("text-editing", _("Text editing")): [
```